### DISCUSSION TITLE
Missing "cap_add: NET_ADMIN" in the Pi Hole part of the docker-compose.yml

### DISCUSSION THREAD

**Discussion by TJB88** [wg-easy/wg-easy#1370] [General]
Noob here.
I could not get Wireguard working with Pi Hole.
In the Pi Hole part of the docker-compose.yml on the following [page](https://github.com/wg-easy/wg-easy/wiki/Using-WireGuard-Easy-with-Pi-Hole), you should add (looks similar to the wg-easy part):

```
    cap_add:
      - NET_ADMIN
```

**Comment by kaaax0815** [comment]
As stated here: https://hub.docker.com/r/pihole/pihole
> Required if you are using Pi-hole as your DHCP server, else not needed
